### PR TITLE
javadoc fixes and cleanup

### PR DESCRIPTION
**Pull request by Raycoms**
# Changes proposed in this pull request:
- Javadoc Error Fixes
- Cleans the Hut Stuff up. Will need a MineColonies change too.
-

Review please

### CODE REVIEW

**Comment by marchermans** [comment]
Why does this still contain shingle stuff?
Version/latest contains 1.18 which should not contain shingle related stuffs ...

**Comment by someaddons** [comment]
version/latest is 1.16 :)

File: build.gradle [hunk 1]
```diff
@@ -75,7 +75,7 @@ dependencies {
     implementation "com.ldtteam:datagenerators:${project.datagenerators_version}"
 }
 
-version = project.minecraft_version + System.getenv().containsKey("Version") ? System.getenv("Version") : 'null'
+version = '0.13.270-ALPHA'
```
Comment (Nightenom) [inline build.gradle]: Huh?

**Review by Nightenom — COMMENTED** [review]
Just asking - why remove html from jdoc?

File: build.gradle (hunk 1 above)
Comment (Raycoms) [reply build.gradle]: sorry - undo

**Comment by marchermans** [comment]
Okey

**Pull request merged**